Client/mods/deathmatch/logic/CResourceModelStreamer.cpp

```diff
@@ -106,7 +106,11 @@ void CResourceModelStreamer::ReleaseAll()
 
 void CResourceModelStreamer::FullyReleaseModel(std::uint16_t modelId)
 {
-    std::uint16_t &refsCount = m_requestedModels[modelId];
```

Comment:
Are you sure that this is the reason? std::unordered_map.operator[] inserts an entry if it doesn't present.

Reply:
But nevertheless there have to be a check, for sure.

Reply:
> std::unordered_map.operator[] inserts an entry if it's not present.

Yes, you are right, but it works only in Debug build. Release build has different behavior here

Reply:
What changes?

Reply:
> > std::unordered_map.operator[] inserts an entry if it's not present.
> 
> Yes, you are right, but it works only in Debug build. Release build has different behavior here

`unordered_map` requires mapped type to be `DefaultConstructible`. That means that a number will always be initialized with zero. There shouldn't be difference between release and debug.

Reply:
I've checked it with breakpoints. For some interesting reason only Debug build is unaffected. I saw similar compiler bugs related to 32 bit applications, but I don't know real reason for this crash.

Reply:
Maybe its related to [the previous `getAccountType` release crash](https://discord.com/channels/801330706252038164/801411628600000522/1244235367406764083)?